### DISCUSSION TITLE
How to get used quota for a nsfs bucket after we set the quota for the bucket

### DISCUSSION THREAD

**Discussion by vamseekrishna25** [noobaa/noobaa-core#6580] [Q&A]
using noobaa api or s3 how can i get

**Comment by nimrod-becker** [comment]
Quota is only supported for data buckets and not namespace buckets

**Reply by vamseekrishna25** [reply]
Okay, can we get the current bucket size in namespace buckets?

**Reply by nimrod-becker** [reply]
No, Since we don't manage the MD for the objects on that bucket, and don't want to do a continuous and cyclic scanning, We don't have that metric available for NS buckets.

I guess specifically for NS FS, since, unlike an AWS bucket, the directory can return the size of everything under it we could implement specifically for NS FS buckets. That won't be as part of the first drop for this feature though.

**Reply by guymguym** [reply]
NSFS also requires scanning for du

**Reply by nimrod-becker** [reply]
that's if its done on the noobaa side, isn't there an offload option? I mean, dependent ofc on the underlying FS and if it supports it in a way that is not scanning all the subtree
